### DISCUSSION TITLE
Specify a colour per meeting in the calendar

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15339] [How to]
Hi guys,

I would like to see how I can specify a colour per meeting created in the calendar? I would like to highlight new customers or existing customer meetings in the calendar using different colours.

Please, could you help me with this?

Thank you
Oscar

**Comment by rskrzypczak** [comment]
You only have this at your disposal:
![image](https://user-images.githubusercontent.com/10221999/137886453-fa649d3a-e0a1-4527-9054-c827a68dac19.png)

**Reply by OscarGS86** [reply]
Thanks @rskrzypczak ,

How Could I edit the colour attached to the events (Call, Meeting, Task) when you create a calendar event? Could I change the name too?

Thank you

**Reply by mariuszkrzaczkowski** [reply]
![image](https://user-images.githubusercontent.com/10198654/137891228-03ca42b1-ff43-416a-ab98-c5ce3d5f9cb6.png)
